src/main/java/edu/kis/powp/command/SetPositionCommand.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class SetPositionCommand implements DriverCommand {
+    private int x;
+    private int y;
+    private Job2dDriver driver;
```

Comment:
Consequently, setters should be applied to other classes in DriverCommand hierarchy.